HowToFIDO.md

```diff
@@ -336,21 +344,24 @@ presumably with a different account. In this case, you should also give
 the user the ability to completely remove their account from being
 listed on the sign-in page.
 
-If the user clicks on “Next”, then check whether you have a credential
-id associated with the user and device (for example, check a cookie or
-read from local storage). If no credential id is available, serve a
+If the user clicks on “Next”, then check whether you have `2FACapableFIDOCredsAvailable` flag associated with the user (for example, check a cookie or
+read from local storage). If no `2FACapableFIDOCredsAvailable` flag is available, serve a
 traditional login challenge suitable for reauthentication, for example:
 
 ![Password-based re-sign-in](images/image3.png)
 
-If, however, you *do* find a credential id for the current session, then
-you can use FIDO-based reauthentication:
+If, however, you *do* have `2FACapableFIDOCredsAvailable` flag for the current session,
+then you can use FIDO-based reauthentication.
+Fetch all `2FACapableFidoCredential` credentials from the server for this user.
+Don't fetch non `2FACapableFidoCredential` credentials like U2F credentials.
+Prefer using long lived local session key while fetching `2FACapableFidoCredential` credentials from the server.
+Also order the credentials in the reverse order of when credentials are created (latest created credential is first in the list):
```

Comment:
Same comments as above: I think this needs to be stronger that `Prefer`, and consider the ordering algorithm.